### DISCUSSION TITLE
Can we refer to another file/slide in current slide

### DISCUSSION THREAD

**Discussion by jd-solanki** [slidevjs/slidev#441] [Q&A]
Let's say I am showing a presentation for my `layouts.md` and I have separate slides for styles in `styles.md` then, can I Provide a link to that `styles.md`'s particular slide in `layouts.md` file?

**Comment by fguisso** [comment]
yes
e.g:

`slide.md`

```markdown
---
# Hello world!
Lorem ipsum dolor
---
src: ../another_dir/another_slide.md
---
Lorem ipsum
```
